src/main/kotlin/no/nav/familie/ef/sak/oppgave/Oppgave.kt

```diff
@@ -14,6 +15,7 @@ data class Oppgave(
     val behandlingId: UUID,
     val gsakOppgaveId: Long,
     val type: Oppgavetype,
+    val vurderHenvendelseOppgavetype: VurderHenvendelseOppgavetype? = null,
```

Comment:
Kunne det vært aktuelt å trenge dette i andre tilfeller hvor det er forskjellige type oppgaver innad i en oppgavetype? Tenkte på om det heller kunne vært et navn ala subtype

Reply:
Enig, det er et litt misvisende navn. Det ser ut som at det er en "vurder henvendelse oppgave", men så er det egentlig en subtype av det. "VurderHenvendelseOppgaveSubtype" e.l ? 🤔

Reply:
Jeg liker "subtype" navngivning. Mindre rom for missforståelser. 

PS: dbskript ikke kjørt i preprod pga. ktint-feil, så fortsatt mulig å endre enkelt.

Reply:
Jeg er enig i at navnet burde være mer generelt. Sånn som navnet er nå er det litt for spesifikt til at man burde feste dette feltet til Oppgave-objektet. 

Kan være enig i at subtype er bedre, eventuelt oppgaveSubtype, siden flere oppgavetyper har flere subtyper igjen.

Reply:
Ser ut som at "oppgaveSubtype" passer fint. Endrer navn på typen og kolonnen. Legger enumen under "oppgave". Høres det greit ut ?